mock-core-configs/etc/mock/templates/centos-stream-9.tpl

```diff
@@ -0,0 +1,45 @@
+config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config redhat-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep'
+config_opts['dist'] = 'el9'  # only useful for --resultdir variable subst
+config_opts['releasever'] = '9'
+config_opts['package_manager'] = 'dnf'
+config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
+# TODO: flip to 'stream9' tag once available
+config_opts['bootstrap_image'] = 'quay.io/centos/centos:stream9-development'
+
+config_opts['dnf.conf'] = """
+[main]
+keepcache=1
+debuglevel=2
+reposdir=/dev/null
+logfile=/var/log/yum.log
+retries=20
+obsoletes=1
+gpgcheck=0
+assumeyes=1
+syslog_ident=mock
+syslog_device=
+mdpolicy=group:primary
+best=1
+protected_packages=
+module_platform_id=platform:el9
+user_agent={{ user_agent }}
+
+[baseos]
+name=CentOS Stream $releasever - BaseOS
+baseurl=https://composes.stream.centos.org/production/latest-CentOS-Stream/compose/BaseOS/$basearch/os/
+failovermethod=priority
+gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-Official
```

Comment:
Another thing I just now thought of, we should remove the `gpgkey` lines and add an explicit `gpg_check=0` line for this initial config.

Reply:
There's gpgcheck=0 set globally, which will need a fix later on once the content is signed.
Do you expect there will be a different key than the CentOS Official?

Reply:
Nope, it should be the same key.  I'm just concerned if someone changes that global setting that this will error out without the explicit `gpgcheck=0`.  It's even more misleading if the gpgkey is already defined.